app/tools/android/make_apk.py

```diff
@@ -320,10 +320,6 @@ def CopyCompressedLibrary(native_path, library_path, raw_path, arch):
 
 
 def CopyNativeLibrary(native_path, library_path, raw_path, arch):
-  dummy_library = os.path.join(native_path, DUMMY_LIBRARY);
-  if os.path.isfile(dummy_library):
-    os.remove(dummy_library)
```

Comment:
@heke123 the dummy library is to ensure correct architecture information when user apk upload to Goggle Play. Please ensure your change does not break that case.